src/CoreBundle/Entity/Course.php

```diff
@@ -782,6 +782,11 @@ public function getCategories()
         return $this->categories;
     }
 
+    public function addCategory(CourseCategory $category)
```

Comment:
Se debe incluir "return $this"; y también hacer que la función regrese el objecto self:

` public function addCategory(CourseCategory $category): self`